src/server/response.cpp

```diff
@@ -97,11 +97,13 @@ std::unique_ptr<Response> Response::build_404(const InternalServer& server, cons
 {
   MustacheData results;
   results.set("url", request.get_full_url());
-
+  // Get URL slug and Name of the book and pass to results
+  std::string url_slug=request.get_full_url().substr((request.get_full_url().find_last_of('/'))+1);
```

Comment:
request already have method to get a part of a url.

Reply:
are you speaking of `std::string RequestContext::get_url_part(int number)` ? 
I did not use this method because the url path might extend to multiple `'/'s` and I require the slug only. I felt using this simple `find_last_of('/')` was just as easy.